frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx

```diff
@@ -59,6 +59,22 @@ const TeamEventCreditForm: React.FC = () => {
   };
 
   const submitTeamEventCredit = async () => {
+    const getCredits: (attendance: TeamEventAttendance[]) => number = (attendance) => {
+      const filteredAttendance = attendance.filter((event) => event.eventUuid == teamEvent?.uuid);
+      const sum = filteredAttendance.reduce(
```

Comment:
reduce 🤩

Reply:
I used reduce at first, but then `event.eventUuid === teamEvent?.uuid` always resulted in false even though they are the same string. Not sure why but using filter resolved that issue.

Reply:
oh mb mb i don't think you need to using `reduce` where the `filter` is, I'm just happy to see the `reduce` on 64 :D